### PR TITLE
Improvements for documentaion

### PR DESCRIPTION
**Pull request by aalmiramolla**
There's no related ticket.

### CODE REVIEW

**Review by serfer2 — APPROVED** [review]

**Comment by aalmiramolla** [comment]
@davidjnevin @isabella-karabasz Can you check the writing as you are English native? Thanks!

File: docs/contribute/index.md [hunk 1]
```diff
@@ -1 +1,30 @@
-# Hello
+# Hello contributor!
+
+In this section, we're gone to explain how to contribute to this project and some guides.
```
Comment (davidjnevin) [inline docs/contribute/index.md:3]: In this section, we're going to explain how to contribute to this project and some guides.

File: docs/contribute/index.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+
+## How can you contribute?
+
+There are many ways to contribute to an Open Source project, as:
+
+* Testing the project and opening Issues.
+* Writing documentation.
+* Collaborate writing code for the project.
+* Making videos, tutorials, etc. of how to use this project.
+* Do Talks to show the project to other people.
```
Comment (davidjnevin) [inline docs/contribute/index.md:16]: Give talks to show the project to other people.

File: docs/contribute/index.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+* And a lot of things more...
```
Comment (davidjnevin) [inline docs/contribute/index.md:17]: And many more things...

File: docs/contribute/index.md [hunk 4]
```diff
@@ -1 +1,30 @@
...
+
+
+### Opening Issues
+
+### Writing documentation
+
+Feel free to open a Pull Request with the changes that you consider using the `Edit` button in the page you want to change and we review it.
```
Comment (davidjnevin) [inline docs/contribute/index.md:24]: To edit a document select the `Edit` button. This takes you to the code on the repository. Make your Pull request and we will review it.

**Comment by aalmiramolla** [comment]
Thanks @davidjnevin for the comments, I fixed it in https://github.com/pythonalicante/MeetupSelector/pull/35/commits/cb4ceed2d635a0492a5ddfbf4a49852c16f2b0f6 commit.

**Pull request merged**